client/src/components/layout/TripsList.js

```diff
@@ -27,6 +27,7 @@ const TripsList = (props) => {
     return (
       <TripTile
         key={trip.id}
+        id={trip.id}
```

Comment:
Instead of separating out each property from the `trip` we can just pass the entire `trip` object as a prop
```
<TripTile
  key={trip.id}
  trip={trip}
/>
```